homeassistant/components/flipr/config_flow.py

```diff
@@ -0,0 +1,95 @@
+"""Config flow for Flipr integration."""
+import logging
+from typing import List
+
+from flipr_api import FliprAPIRestClient
+from requests.exceptions import HTTPError, Timeout
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+
+from .const import CONF_FLIPR_IDS
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Flipr."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    def __init__(self):
+        """Initialize Flipr config flow."""
+        self._username = None
+        self._password = None
+        self._flipr_ids = None
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+        if user_input is None:
+            return self._show_setup_form()
+
+        self._username = user_input[CONF_EMAIL]
+        self._password = user_input[CONF_PASSWORD]
+
+        if not self._flipr_ids:
+            try:
+                flipr_ids = await self._authenticate_and_search_flipr()
+            except HTTPError:
+                errors = {}
+                errors["base"] = "invalid_auth"
+                return self._show_setup_form(errors)
+            except (Timeout, ConnectionError):
+                errors = {}
+                errors["base"] = "cannot_connect"
+                return self._show_setup_form(errors)
+            except Exception as exception:  # pylint: disable=broad-except
+                errors = {}
+                errors["base"] = "unknown"
+                _LOGGER.exception(exception)
+                return self._show_setup_form(errors)
+
+            if len(flipr_ids) == 0:
+                # No flipr_id found. Tell the user with an error message.
+                errors = {}
+                errors["base"] = "no_flipr_id_found"
+                return self._show_setup_form(errors)
+
+            # If multiple flipr found (rare case), we concatene the ids to create multiple devices in this configuration.
```

Comment:
```suggestion
            # If multiple flipr found (rare case), we concatenate the ids to create multiple devices in this configuration.
```

Additionally, why not store a list instead of concatinating?

Reply:
I considered it and managed to change it to store a list.